net/interfaces/interfaces.go

```diff
@@ -474,7 +474,7 @@ func hasTailscaleIP(pfxs []netaddr.IPPrefix) bool {
 	return false
 }
 
-func isTailscaleInterface(name string, ips []netaddr.IPPrefix) bool {
+func IsTailscaleInterface(name string, ips []netaddr.IPPrefix) bool {
```

Comment:
exported so needs docs now

Reply:
No longer need to export this now that there's a `HasCGNATInterface` helper method.